Common/ConvertUproot.py

```diff
@@ -0,0 +1,47 @@
+import uproot
+import awkward as ak
+import os
+import numpy as np
+def saveFile(outFile, out_tree, treeName):
+  if os.path.exists(outFile):
+    os.remove(outFile)
+    with uproot.recreate(outFile, compression=uproot.LZMA(9)) as out_file:
+      out_file[treeName] = out_tree
+      out_file.close()
+    return out_file,outFile
```

Comment:
`out_file` is closed at this point. you shouldn't return it.
```suggestion
return outFile
```

Reply:
ok